### DISCUSSION TITLE
What is the difference between 'Usage Policy' and License?

### DISCUSSION THREAD

**Discussion by matgnt** [International-Data-Spaces-Association/ids-specification#154] [Q&A] [answered]
Since Usage Policies is the 'new' part of what IDS is capable of doing, I think it needs to be explained how it relates to 'licenses' like e.g. 'Creative Commons' and other, also proprietary licenses (paper contracts).

Opinions on that?

**Comment by PeterKoen-MSFT** [comment, marked as answer]
Licenses are a legal aspect and therefore part of the legal layer of a dataspace. However, a usage policy can reference a license or even enforce a license.
If a data plane is used where the enforcement of the license is technically not possible or insufficient the obligation to follow the license agreement in the legal domain is still to be followed. Usage policies should not overrule the licensing agreement. It is in the responsibility of both parties to check and adhere to legal agreements.
This is no different from data sharing agreements/licensing agreements without dataspaces. If one party violates the license there will be legal consequences.

**Comment by gbrost** [comment]
I totally agree: Usage policies are technical means to enforce or, if not possible, at least document usage agreements. They can refer to a license, but are not necessarily the same.
